### DISCUSSION TITLE
How to use patches?

### DISCUSSION THREAD

**Discussion by Jaceideu** [MCPHackers/RetroMCP-Java#145] [Q&A] [answered]
Sorry for another noob question but wiki is not much of help. I've created .patch file using RetroMCP. How do you use it?

**Comment by PhoenixVX** [comment]
Throw it in the conf folder before decompiling.

**Comment by Lassebq** [comment, marked as answer]
Or use apply patch task. In GUI hidden under MCP -> More Tasks
It uses patches/client.patch or patches/server.patch under your current RetroMCP workspace
